test/dev.ts

```diff
@@ -94,16 +93,3 @@ process.env.PAYLOAD_DROP_DATABASE = process.env.PAYLOAD_DROP_DATABASE === 'false
 // fetch the admin url to force a render
 void fetch(`http://localhost:${port}${adminRoute}`)
 void fetch(`http://localhost:${port}/api/access`)
-// This ensures that the next-server process is killed when this process is killed and doesn't linger around.
-process.on('SIGINT', () => {
-  if (child) {
-    child.kill('SIGINT')
-  }
-  process.exit(0)
-})
-process.on('SIGTERM', () => {
-  if (child) {
-    child.kill('SIGINT')
-  }
-  process.exit(0) // Exit the parent process
-})
```

Comment:
@AlessioGr this threw me off because I could see that `child` was exported from safelyRunScript and assigned there in a function that I don't think was ever being called.

I need to know how this was supposed to work or risk having a regression.